Sources/SwiftSentry/SentrySDK.swift

```diff
@@ -134,11 +134,81 @@ public enum SentrySDK {
     // This is safe to be called from a crashing thread and may not return.
     public static func captureExceptionRecord(exceptionRecord: UnsafeMutablePointer<EXCEPTION_POINTERS>) {
         var exceptionContext = sentry_ucontext_s()
+        // Use the custom `captureStowedExceptions` reporter if the exception code is the stowed exception code. This
+        // will include more information about the crash.
+        guard let record = exceptionRecord.pointee.ExceptionRecord else {
+            let breadcrumb = sentry_value_new_breadcrumb("Empty exception record", "ERROR: The exception record is empty")
+            sentry_add_breadcrumb(breadcrumb)
+            return
+        }
+
+        let stowedExceptionCode = 0xC000027B
+        if record.pointee.ExceptionCode == stowedExceptionCode {
+            captureStowedExceptions(exceptionRecord: record.pointee)
+        } else {
+            exceptionContext.exception_ptrs = exceptionRecord.pointee
+            withUnsafePointer(to: &exceptionContext) { exceptionContextPtr in
+                sentry_handle_exception(exceptionContextPtr)
+            }
+        }
+    }
+
+    private static func captureStowedExceptions(exceptionRecord: EXCEPTION_RECORD) {
+        let event = Event(level: SentryLevel.fatal)
+        event.message = "This is a crash with stowed exceptions. The events are grouped by the stack trace of the latest stowed exception.\n" +
+                        "You can find the crash stack of the other stowed exceptions and of the outer crash by scrolling down."
+        let eventSerialized = event.serialized()
 
-        exceptionContext.exception_ptrs = exceptionRecord.pointee
-        withUnsafePointer(to: &exceptionContext) { exceptionContextPtr in
-            sentry_handle_exception(exceptionContextPtr)
+        // Log the outer crash stack trace and all the stowed exceptions as distinct exception events.
+        // The events will be displayed in the Sentry UI as a single event with multiple stack traces.
+        let exceptions = sentry_value_new_list();
+        let exception = sentry_value_new_exception("Outer crash", "Outer crash with stowed exceptions")
+        sentry_value_set_stacktrace(exception, nil, 0)
+        sentry_value_append(exceptions, exception);
+        sentry_value_set_by_key(eventSerialized, "exception", exceptions);
+
+        let exceptionInfo = exceptionRecord.ExceptionInformation
+        // For stowed exceptions, the first element in `ExceptionInformation` is a pointer to an array of `STOWED_EXCEPTION_INFORMATION_V2`
+        // and the second element is the total number of stowed exceptions in this array
+        if let arrayPointer = UnsafeMutablePointer<UnsafeMutablePointer<STOWED_EXCEPTION_INFORMATION_V2>?>(bitPattern: UInt(exceptionInfo.0)) {
```

Comment:
i don't think swift guarantees memory layout for this to always be safe, would be better to use the C structure if we can